docs/SERVER_CONFIGURATIONS.md

```diff
@@ -278,33 +279,37 @@ LHS_LISTENER_MY_LISTENER_CA_CERT=/tmp/ca
 
 ---
 
-### `LHS_LISTENERS_AUTHORIZATION_MAP`
+### `LHS_LISTENERS_AUTHENTICATION_MAP`
 
-Map between listener names and authorization protocols. Depends on `LHS_LISTENERS`.
+Map between listener names and authentication mechanisms. Depends on `LHS_LISTENERS`.
+
+> `LHS_LISTENERS_PROTOCOL_MAP` should be set to `MTLS` for a listener in order to use `MTLS` as authentication mechanisms.
 
 Format: `<LISTENER NAME>:<AUTH PROTOCOL>`
 
 Examples of a legal list: `SECURE:OAUTH,INSECURE:NONE`
 
-Allowed protocols: `NONE`, `OAUTH`
+Allowed protocols: `NONE`, `OAUTH`, `MTLS`
 
 - **Type:** map
 - **Default:** null
 - **Importance:** high
 
 ---
 
-### `LHS_LISTENER_<LISTENER NAME>_AUTHORIZATION_SERVER`
+### `LHS_OAUTH_SERVER_URL`
+
+Optional OAuth server URL. Used by the server to identify the OpenId Connect endpoints and validate incoming calls.
```

Comment:
Should we say "authenticate" instead of "validate"?

Additionally, this is probably a question for Saúl, but are you sure OIDC is a thing here? I thought OIDC involved human flows and was something built on top of OAuth to communicate information about what groups a user belongs to.